test/uk/gov/hmrc/plasticpackagingtax/registration/models/registration/RegistrationSpec.scala

```diff
@@ -55,10 +55,10 @@ class RegistrationSpec
         notStartedRegistration.numberOfCompletedSections mustBe 0
 
         notStartedRegistration.isCompanyDetailsComplete mustBe false
-        notStartedRegistration.companyDetailsStatus mustBe TaskStatus.NotStarted
+        notStartedRegistration.companyDetailsStatus mustBe TaskStatus.CannotStartYet
 
         notStartedRegistration.isLiabilityDetailsComplete mustBe false
-        notStartedRegistration.liabilityDetailsStatus mustBe TaskStatus.CannotStartYet
+        notStartedRegistration.liabilityDetailsStatus mustBe TaskStatus.NotStarted
```

Comment:
It probably makes sense to reorder these checks to reflect the on-page order.

Reply:
Done!

Reply:
I actually meant the companyDetails, liabilityDetails checks should be made in the same order as the sections on the page but don't worry it should not hold up a merge.